src/room.cpp

```diff
@@ -78,6 +78,27 @@ void Room::HandleJoinRequest(const RakNet::Packet* packet) {
     SendJoinSuccess(member);
 }
 
+void Room::HandleChatPacket(const RakNet::Packet* packet) {
+    RakNet::BitStream in_stream(packet->data, packet->length, false);
+
+    in_stream.IgnoreBytes(sizeof(RakNet::MessageID));
+    RakNet::RakString message;
+    in_stream.Read(message);
+    const auto sending_member = std::find_if(members.begin(), members.end(),
+                                            [&](const Member member) {return member.network_address == packet->systemAddress;});
```

Comment:
huh.. what Subv meant with the newline commit was: keep the lambda variable but format it differently:

```cpp
auto whatEverTheLambdaNameWas = [&](const Member& member) -> bool {
  return member.network_address == packet->systemAddress;
});
```

or something